### PR TITLE
Add maipet

### LINKED ISSUE #519: Add  @maipet to team
@maipet is missing in our team description atm http://lobid.org/team/.

### PR DESCRIPTION
**Pull request by dr0i**
Resolves  #519.

### CODE REVIEW

**Comment by dr0i** [comment]
Please review @maipet (and add data if you like) . Have a look at http://stage.lobid.org/team/pm/#! and http://stage.lobid.org/team-de/.

**Comment by fsteeg** [comment]
I think for products/projects we should add:

- lobid-resources
- oersi
- rpb

File: gatsby/lobid/static/team.json [hunk 1]
```diff
@@ -187,6 +187,17 @@
         "id": "http://lobid.org/team/pt#!"
       }
     },
+    {
+      "type": "Role",
+      "startDate": "2023",
+      "roleName": {
+        "de": "Bibliothekar",
```
Comment (fsteeg) [inline gatsby/lobid/static/team.json:194]:
```suggestion
        "de": "Bibliothekarin",
```

**Comment by fsteeg** [comment]
Updated and deployed to stage:

http://stage.lobid.org/team/pm & http://stage.lobid.org/team

@maipet Let me know if you want to add a picture or additional links, or just approved as is and I will deploy to production.

**Comment by maipet** [comment]
@fsteeg maybe an additional sameAs: https://github.com/maipet, thx

**Comment by fsteeg** [comment]
Deployed to production:

https://lobid.org/team/pm & https://lobid.org/team

**Pull request merged**